examples/form-handler/handlers/Input.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react'
+import { FormGroup, ControlLabel, FormControl } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+import { inputChange } from '../actions'
+
+class Input extends Component {
+  inputChange = (title, name, e) => {
+    this.props.inputChange(title, name, e.target.value)
+  };
+
+  render () {
+    return (
+      <div>
+        <FormGroup controlId='formBasicText'>
+          <ControlLabel>{this.props.controlLabel}</ControlLabel>
+          <FormControl
+            disabled={this.props.disabled}
+            type={this.props.type || 'Text'}
+            placeholder={this.props.controlLabel}
+            onChange={(e) => this.inputChange(this.props.title, this.props.name, e)}
```

Comment:
maybe
 `onChange={this.inputChange}`
and
`inputChange = (e) => this.props.inputChange(this.props.title, this.props.name, e.target.value) `?

Reply:
The changes has been made

Reply:
@estrada9166 
if you like destructuring, it also can be smth like this:
```
inputChange = (e) => {
 const { 
  inputChange,
  title,
  name,
 } = this.props;

 inputChange(title, name, e.target.value);
}
```

Reply:
Done! Thx